apps/spark/src/test/java/com/linkedin/openhouse/jobs/spark/OperationsTest.java

```diff
@@ -186,15 +187,21 @@ public void testOrphanFilesDeletionJavaAPI() throws Exception {
       DeleteOrphanFiles.Result result =
           ops.deleteOrphanFiles(table, TRASH_DIR, System.currentTimeMillis(), false);
       List<String> orphanFiles = Lists.newArrayList(result.orphanFileLocations().iterator());
-      log.info("Detected {} orphan files", orphanFiles.size());
-      for (String of : orphanFiles) {
+      List<String> orphanFilesWithoutMetadata =
+          orphanFiles.stream()
```

Comment:
why metadata.json files filtered out?

Reply:
Seems that the tests were meant for locations without schema: [See this slack message from Stas](https://linkedin-randd.slack.com/archives/C02Q94UU6TE/p1720562662594949?thread_ts=1720556484.877769&cid=C02Q94UU6TE)